packages/react/src/CartProvider.test.helpers.ts

```diff
@@ -81,3 +81,18 @@ export function getCartLineMock(
 ): PartialDeep<CartLine, {recurseIntoArrays: true}> {
   return {...CART_LINE, ...options};
 }
+
+export const getCartActionsMock = () => ({
```

Comment:
Hm, is this something that will automatically be mocked out if you do `vi.mock()` in the test file instead? Seems like that would maybe be better than trying to keep this mocked file up-to-date with the shape of the api and stuff?

Reply:
All good! I originally sorta added this to appease the types, but found another way to get around that.